src/lipalightninglib.udl

```diff
@@ -539,3 +544,442 @@ enum PayErrorCode {
 
     "UnexpectedError", // An unexpected error occurred. This likely is a result of a bug within 3L/LDK and should be reported to lipa.
 };
+
+enum CountryCode {
```

Comment:
Hmmm... this large enums are kind of ugly. But I can't come up with any better way right now.

How about if we don't separate them by new lines for once to save some scrolling space?

Also, the first enum is sorted alphabetically, while the second isn't. What's the reason for that?

Separated by <space> instead of <new line> and sorted alphabetically if you need it:

`"AFG", "ALA", "ALB", "DZA", "ASM", "AND", "AGO", "AIA", "ATA", "ATG", "ARG", "ARM", "ABW", "AUS", "AUT", "AZE", "BHS", "BHR", "BGD", "BRB", "BLR", "BEL", "BLZ", "BEN", "BMU", "BTN", "BOL", "BES", "BIH", "BWA", "BVT", "BRA", "IOT", "BRN", "BGR", "BFA", "BDI", "CPV", "KHM", "CMR", "CAN", "CYM", "CAF", "TCD", "CHL", "CHN", "CXR", "CCK", "COL", "COM", "COG", "COD", "COK", "CRI", "CIV", "HRV", "CUB", "CUW", "CYP", "CZE", "DNK", "DJI", "DMA", "DOM", "ECU", "EGY", "SLV", "GNQ", "ERI", "EST", "ETH", "FLK", "FRO", "FJI", "FIN", "FRA", "GUF", "PYF", "ATF", "GAB", "GMB", "GEO", "DEU", "GHA", "GIB", "GRC", "GRL", "GRD", "GLP", "GUM", "GTM", "GGY", "GIN", "GNB", "GUY", "HTI", "HMD", "VAT", "HND", "HKG", "HUN", "ISL", "IND", "IDN", "IRN", "IRQ", "IRL", "IMN", "ISR", "ITA", "JAM", "JPN", "JEY", "JOR", "KAZ", "KEN", "KIR", "PRK", "KOR", "KWT", "KGZ", "LAO", "LVA", "LBN", "LSO", "LBR", "LBY", "LIE", "LTU", "LUX", "MAC", "MKD", "MDG", "MWI", "MYS", "MDV", "MLI", "MLT", "MHL", "MTQ", "MRT", "MUS", "MYT", "MEX", "FSM", "MDA", "MCO", "MNG", "MNE", "MSR", "MAR", "MOZ", "MMR", "NAM", "NRU", "NPL", "NLD", "NCL", "NZL", "NIC", "NER", "NGA", "NIU", "NFK", "MNP", "NOR", "OMN", "PAK", "PLW", "PSE", "PAN", "PNG", "PRY", "PER", "PHL", "PCN", "POL", "PRT", "PRI", "QAT", "REU", "ROU", "RUS", "RWA", "BLM", "SHN", "KNA", "LCA", "MAF", "SPM", "VCT", "WSM", "SMR", "STP", "SAU", "SEN", "SRB", "SYC", "SLE", "SGP", "SXM", "SVK", "SVN", "SLB", "SOM", "ZAF", "SGS", "SSD", "ESP", "LKA", "SDN", "SUR", "SJM", "SWZ", "SWE", "CHE", "SYR", "TWN", "TJK", "TZA", "THA", "TLS", "TGO", "TKL", "TON", "TTO", "TUN", "TUR", "TKM", "TCA", "TUV", "UGA", "UKR", "ARE", "GBR", "USA", "UMI", "URY", "UZB", "VUT", "VEN", "VNM", "VGB", "VIR", "WLF", "ESH", "YEM", "ZMB", "ZWE"`

`"Aa", "Ab", "Ae", "Af", "Ak", "Am", "An", "Ar", "As", "Av", "Ay", "Az", "Ba", "Bh", "Bm", "Bn", "Bo", "Bs", "Bg", "Br", "Ca", "Ce", "Ch", "Co", "Cr", "Cs", "Cu", "Cy", "Cv", "Da", "De", "Dv", "Dz", "Ee", "El", "En", "Eo", "Es", "Et", "Eu", "Fa", "Ff", "Fi", "Fj", "Fo", "Fr", "Fy", "Ga", "Gd", "Gl", "Gn", "Gu", "Gv", "Ha", "He", "Hi", "Ho", "Hr", "Ht", "Hz", "Hy", "Id", "Ia", "Ie", "Ig", "Ik", "Io", "Is", "It", "Ii", "Ja", "Jv", "Ka", "Kg", "Ki", "Kj", "Kk", "Kl", "Kn", "Ko", "Kr", "Ks", "Ku", "Kv", "Kw", "Ky", "La", "Lb", "Lg", "Li", "Ln", "Lo", "Lt", "Lu", "Lv", "Mg", "Mh", "Mi", "Mk", "Ml", "Mn", "Mt", "Ms", "My", "Na", "Nd", "Ne", "Ng", "Nl", "Nn", "No", "Nr", "Ns", "Ny", "Oc", "Oj", "Om", "Or", "Os", "Pa", "Pi", "Pl", "Ps", "Pt", "Qu", "Rm", "Rn", "Ro", "Ru", "Rw", "Sa", "Sc", "Sd", "Se", "Sg", "Sh", "Si", "Sk", "Sl", "Sm", "Sn", "So", "Sq", "Sr", "Ss", "St", "Su", "Sv", "Sw", "Ta", "Te", "Tg", "Th", "Ti", "Tk", "Tl", "Tn", "To", "Tr", "Ts", "Tt", "Tw", "Ty", "Ug", "Uk", "Ur", "Us", "Uz", "Ve", "Vi", "Vo", "Wa", "Wo", "Xh", "Yi", "Yo", "Za", "Zh", "Zu"`